analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -33,6 +33,9 @@ public override bool ShouldExecute()
         return walker.Result;
     }
 
+    protected override bool IsDisposeMethod(IMethodSymbol method) =>
```

Comment:
Can be simplified, because `IsDisposeMethod(method)` reads as "is dispose method method"
```suggestion
    protected override bool IsDispose(IMethodSymbol method) =>
```